configman/config_manager.py

```diff
@@ -675,7 +686,9 @@ def _overlay_expand(self):
                 if key in all_reference_values:
                     # make sure that this value gets propagated to keys
                     # even if the keys have already been overlaid
-                    known_keys -= set(all_reference_values[key])
+                    finished_keys -= set(
+                        all_reference_values[key]
+                    )
```

Comment:
Doing 
```python
for k in mydict:
    print mydict[k]
```
is the exact same thing as:
```python
for values in mydict.values():
    print values
```
I know you know that but I just wanted to be clear.